### PR TITLE
WIFI-1552 Address auto/manual changes to backup channel number

### PR DESCRIPTION
**Pull request by lynnshi-design**

### CODE REVIEW

File: opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java [hunk 1]
```diff
@@ -68,6 +68,11 @@ void configureWifiRrm(OvsdbClient ovsdbClient, OpensyncAPConfig opensyncApConfig
...
                 continue; // don't have a radio of this kind in the map
             }
+            boolean autoChannelSelection = rfElementConfig.getAutoChannelSelection();
+            int backupChannel = elementRadioConfig.getActiveBackupChannel(autoChannelSelection);
+            LOG.debug("configureWifiRadios autoChannelSelection {} activeBackupChannel {}",
+                    autoChannelSelection, backupChannel);
+            
             AutoOrManualValue probeResponseThresholdDb = null;
             AutoOrManualValue clientDisconnectThresholdDb = null;
             if (elementRadioConfig != null && rfElementConfig != null) {
```
Comment (dmitry-toptygin-connectus-ai) [inline opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java:78]: this check seems to be redundant - with what's happening on line 68. although it is not part of current changes...

**Review by dmitry-toptygin-connectus-ai — APPROVED** [review]

File: opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java (hunk 1 above)
Comment (mikehansen1970) [reply opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java:78]: Believe it is leftover from the period when we didn't have the configuration maps for the radios in the config maps. There was a brief period when the auto provision was initially disabled for some APs and some of these values were coming up null. It is probably safe to remove it now.

**Review by mikehansen1970 — APPROVED** [review]
While I still have a problem with the concept of backup and manual backup channels and that approach to configuration, I see nothing in the code itself that concerns me from an implementation perspective.

File: opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java (hunk 1 above)
Comment (dmitry-toptygin-connectus-ai) [reply opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java:76]: well, the way it is now is worse.
(original) lines 68-70 did the null checks, and I would say they were doing a good job at it.
(new) line 76 was redundant, and that's where I commented.
the way things are right now, (new) lines 69 and 70 can throw an NPE.

Comment (lynnshi-design) [reply opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java]: I was puzzled for the change myself as well. Now I get it, thank you for clarification.

**Review by dmitry-toptygin-connectus-ai — APPROVED** [review]

**Pull request merged**